README.md

```diff
@@ -25,13 +25,13 @@ $ export API_ENDPOINT='https://cells.dev.hubmapconsortium.org/api/'
 >>> [m for m in dir(client) if m.startswith('select_')]
 ['select_cells', 'select_clusters', 'select_datasets', 'select_genes', 'select_organs', 'select_proteins']
 
->>> gene_symbol = client.select_genes().get_list()[0]['gene_symbol']
+>>> gene_symbol = client.select_genes(where="modality", has=["rna"]).get_list()[0]['gene_symbol']
```

Comment:
Can you explain the behavior change? Does the underlying API now require constraints? or is the shape of response records not consistent? or something else?

Reply:
This is the same issue as https://github.com/hubmapconsortium/portal-ui/issues/2584. I believe this is still easy for users to trip over. Put yourself in their shoes.